.circleci/config.yml

```diff
@@ -6,6 +6,18 @@ common: &common
   working_directory: ~/repo
   steps:
     - checkout
+    - run:
+        name: merge pull request base
+        command: |
+          if [[ -n "${CIRCLE_PR_NUMBER}" ]]; then
+            PR_INFO_URL=https://api.github.com/repos/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME/pulls/$CIRCLE_PR_NUMBER
+            curl -L "$PR_INFO_URL" > /tmp/pr-info.json
+
+            PR_BASE_BRANCH=$(python -c 'import json, sys; obj = json.loads(open("/tmp/pr-info.json").read()); sys.stdout.write(obj["base"]["ref"])')
```

Comment:
Nice use of `ba-py-json`, the Tur-duck-en of scripting. :)

So `obj["base"]["ref"]` is almost always going to be master, right? I'm just trying to make sure I understand what's happening here.

Reply:
The base branch is chosen by the author of the pull request when they create it from a branch on their fork.  As far as I know, the base branch can be any branch in the upstream repo.  But it defaults to master.